### PR TITLE
Enhancement/drawable graph

### PR DESCRIPTION
**Pull request by Vennik**
- Splitted drawable graph in preparation of collapsing.

### CODE REVIEW

File: src/main/java/nl/tudelft/context/drawable/DrawablePosition.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.value.ObservableDoubleValue;
+
+/**
+ * @author René Vennik <renevennik@gmail.com>
+ * @version 1.0
+ * @since 31-5-2015
+ */
+public class DrawablePosition {
```
Comment (qurben) [inline src/main/java/nl/tudelft/context/drawable/DrawablePosition.java]: Make this `abstract`?

Comment (Vennik) [inline src/main/java/nl/tudelft/context/drawable/DrawablePosition.java]: Newick uses it.

Comment (Vennik) [inline src/main/java/nl/tudelft/context/drawable/DrawablePosition.java]: Done.

**Comment by qurben** [comment]
:shipit:! More classes is more better.

**Pull request merged**